eng/pipelines/templates/steps/cosmos-integration-public.yml

```diff
@@ -3,32 +3,14 @@ parameters:
 
 steps:
   - powershell: |
-      Write-Host "Downloading Cosmos Emulator - ${{ parameters.EmulatorMsiUrl }}"
-      wget  "${{ parameters.EmulatorMsiUrl }}" -outfile "$env:temp\azure-cosmosdb-emulator.msi"
-      Write-Host "Finished Downloading Cosmos Emulator - $env:temp\azure-cosmosdb-emulator.msi"
-      dir "$env:temp"
-    displayName: "Download Public Cosmos DB Emulator"
-
-  - script: |
-      choco install lessmsi
-      choco upgrade lessmsi
-      mkdir "%TEMP%\Azure Cosmos DB Emulator"
-      lessmsi x "%TEMP%\azure-cosmosdb-emulator.msi" "%TEMP%\Azure Cosmos DB Emulator\"
-    displayName: "Install Public Cosmos DB Emulator"
-
-  - powershell: |
-      Write-Host "Starting Comsos DB Emulator"
-      Start-Process "$env:Temp\Azure Cosmos DB Emulator\SourceDir\Azure Cosmos DB Emulator\CosmosDB.Emulator.exe" "/NoExplorer /NoUI" -Verb RunAs
-    displayName: "Run Public Cosmos DB Emulator"
+      $targetDir = $env:temp
+      Write-Host "Downloading and extracting Cosmos DB Emulator - ${{ parameters.EmulatorMsiUrl }}"
+      Write-Host "Target Dir: $targetDir"
+      msiexec /a ${{ parameters.EmulatorMsiUrl }} TARGETDIR=$targetDir /qn | wait-process
+    displayName: Download and Extract Public Cosmos DB Emulator
 
   - powershell: |
-      Import-Module "$env:Temp\Azure Cosmos DB Emulator\SourceDir\Azure Cosmos DB Emulator\PSModules\Microsoft.Azure.CosmosDB.Emulator"
-
-      Do {
-          sleep 5
-          $cosmosStatus = Get-CosmosDbEmulatorStatus
-          Write-Host "Cosmos Status: $cosmosStatus"
-      } While ($cosmosStatus -ne 'Running')
-
-      Write-Host "Done"
-    displayName: "Check Public Cosmos DB Emulator Status"
+      Write-Host "Launching Cosmos DB Emulator"
+      Import-Module "$env:temp\Azure Cosmos DB Emulator\PSModules\Microsoft.Azure.CosmosDB.Emulator"
```

Comment:
Does importing this also handle installing the certificate? IIRC there was a certificate step also needed.

Reply:
Certificate import is not required for JS because SSL validation is disabled for these tests.

Reply:
Ok when you generalize this to the other languages you might need to add a conditional step to install the certificate.